src/component/Context/CartContext/CartContext.js

```diff
@@ -1,61 +1,34 @@
-import { useContext, createContext, useReducer } from "react";
-import { useWishList } from "../../index";
+import { useContext, createContext, useReducer, useEffect } from "react";
```

Comment:
Suggestion : We can extract out the code for reducer in a separate file and functions for api calls (increase quantity,addToCart ,deleteFromCart etc) could be extracted out to a custom hook . Will make the mode more readable .

Reply:
Thanks for the suggestion  abhishek will do the changes in the next PR